src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -8394,11 +8420,13 @@ private void VerifyQuotedInvalid<T>(string name, string value, T expected, Func<
             [Fact]
```

Comment:
That's unfortunate. https://github.com/dotnet/roslyn/issues/12753 states that ```/debug:full``` is OS specific, but not runtime specific. Does Mono not support COM at all? The interop we need is very simple (no COM registration needed). We P/Invoke native dll that directly returns COM object, which we then call methods on.